guides/advanced_guides/datatypes.md

```diff
@@ -0,0 +1,99 @@
+# Data Types
+
+This page describes the different data types supported for the fields in a document and how MeiliSearch does handle them.
+
+- [String](/guides/advanced_guides/datatypes.md#string)
+- [Numeric types](/guides/advanced_guides/datatypes.md#numeric-types): `integer`, `float`
+- [Boolean](/guides/advanced_guides/datatypes.md#boolean)
+- [Array](/guides/advanced_guides/datatypes.md#array)
+- [Object](/guides/advanced_guides/datatypes.md#object)
+- [`null` type](/guides/advanced_guides/datatypes.md#null-type)
+
+When documents are returned upon search, they contain [displayed fields](/guides/advanced_guides/field_properties.md#displayed-fields). The displayed values are always **returned unchanged**.
+
+When a field is [searchable](/guides/advanced_guides/field_properties.md#searchable-fields), each data type is handled in a different way.
+
+### String
+
+> String tokenization is the process of splitting a string into a list of individual terms that are called tokens.
+
+A string is passed to a tokenizer and is then broken into separate string tokens. Each word (token) is indexed and stored in the global dictionary of the corresponding index.
+
+- For Latin-based languages, the words are separated by space.
+- For Kanji characters, the words are separated by character.
+
+There are two kinds of space separators:
+
+- **Soft spaces** (distance: 1): whitespaces, quotes, `'-' | '_' | '\'' | ':' | '/' | '\\' | '@'`
+- **Hard spaces** (distance: 8): `'.' | ';' | ',' | '!' | '?' | '(' | ')'`
+
+#### Example
+
+```
+"Bruce Willis,Vin Diesel"
+```
+
+In the example above, the distance between `Bruce` and `Willis` is equal to **1**. The distance between `Vin` and `Diesel` is equal to **1** too.
+But, the distance between `Bruce` and `Vin` is equal to **8**. The same goes for `Bruce` and `Diesel`, or `Vin` and `Willis`, or also `Willis` and `Diesel`.
```

Comment:
What does this imply for the users?

Does that mean that `Bruce`, `Willis`, `Vin` and `Diesel` will be indexed ? Why is the distance meaningful?

Reply:
You're right. I should rethink the whole guide focusing on what info is important for the users. Maybe I should remove this part about the distance.